### ISSUE TITLE
[Query] 1. Dropout & 2. training in steps vs training in epochs

### ISSUE THREAD

**Issue by agenius5** [google-research/pegasus#154]
Hi, @JingqingZ 

First, why are we training in steps and not epochs? Most of the models I have seen are usually trained in epochs but why not PEGASUS?

Second, you used a dropout of 0.1. Did you experimented with higher dropout rates? Actually, in my experiments I increased the dropout when validation sets performance started to drop (resumed training with increased dropout) and performance improved. So, any insights on it?

**Comment by JingqingZ** [comment]
> First, why are we training in steps and not epochs? Most of the models I have seen are usually trained in epochs but why not PEGASUS?

There was no particular reason of using steps rather than epochs. But since the pre-trained corpus is very large, a base model may not observe one complete epoch.
 
> Second, you used a dropout of 0.1. Did you experimented with higher dropout rates? Actually, in my experiments I increased the dropout when validation sets performance started to drop (resumed training with increased dropout) and performance improved. So, any insights on it?

We didn't try larger dropout. A larger dropout may prevent overfitting.

**Comment by agenius5** [comment]
Okay, Thanks!
